_source/_posts/2024-07-30-Update-to-the-PMHC-MDS-v-2-12-8.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Clarification and Update to the PMHC MDS - 30/7/2024 (v2.12.8)
+---
+
+You may have seen a recent PMHC MDS notification that users from State 
+and Territory Governments will now be given access to aggregate PMHC MDS 

```

Comment:
We actually said "can now be given access", which aligns much better with the rest of this paragraph, being a managed process and not a free-for-all.